### ISSUE TITLE
Attribute with "Catalog Input Type for Store Owner" equal "Fixed Product Tax" for Multi-store

### ISSUE THREAD

**Issue by klakovsky** [magento/magento2#12393]
Attribute with "Catalog Input Type for Store Owner" equal "Fixed Product Tax" did not work correctly for multi-stores.

### Preconditions

1. CE 2.1.9 or CE 2.2.1 or CE 2.3-develop
2. PHP v7.0.22

### Steps to reproduce

1. Create two websites website1 (website_id = 1),  website2 (website_id =2)
2. Create new product SKU=product1 with default attribute set and check for "Product in Websites" only website2. 
3. Create new product attribute "attribute1" with "Catalog Input Type for Store Owner" equal "Fixed Product Tax".
4. Assign attribute1 to default attribute set.
5. Open product "product1" and find attribute1 and click the button "add" near "attribute1".

### Expected result

1. The new line with website column.

### Actual result

1. The new line without website column.
![image](https://user-images.githubusercontent.com/6290121/33127884-6bad838c-cf9b-11e7-8fd9-576327c20256.png)

2. Browser console error: 
```
Uncaught TypeError: data.some is not a function
at findFirst (select.js:67)
at UiClass.clear (select.js:285)
at UiClass.setInitialValue (select.js:299)
at UiClass.setInitialValue (wrapper.js:109)
at UiClass.initialize (abstract.js:70)
at UiClass.initialize (wrapper.js:109)
at UiClass._super (wrapper.js:106)
at UiClass.initialize (select.js:119)
at UiClass.initialize (wrapper.js:109)
at new UiClass (class.js:49)
```

### Comments
The method 
`\Magento\Weee\Ui\DataProvider\Product\Form\Modifier\Manager\Website::getWebsites()`
should return the array without keys like `\Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\Websites::getWebsitesList()`

**Comment by magento-engcom-team** [comment]
@klakovskiy, thank you for your report.
We've created internal ticket(s) MAGETWO-84388 to track progress on the issue.

**Comment by magento-team** [comment]
Hi @klakovskiy. Thank you for your report. 
The issue has been fixed in magento/magento2#13019 by @dverkade in 2.2-develop branch
Related commit(s):
 - 13546a71cd7169549d08546d56daca0859dc73cb

The fix will be available with the upcoming patch release.

**Comment by okorshenko** [comment]
Hi @klakovskiy. Thank you for your report. 
The issue has been fixed in magento/magento2#13020 by @dverkade in 2.1-develop branch
Related commit(s):
 - 31c1939addaca256b33bd88f4c85fcfb7c5d9e3d
 - 2502f849d17334bf7f8c673bb13acd30163a2748

The fix will be available with the upcoming patch release.
